plugins/aws-costs/results.schema

```diff
@@ -0,0 +1,8 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "required": [
+    ],
+    "properties": {
+    }
+}
```

Comment:
I'd love to see the actual schema here. What does the output look like?

This is in http://json-schema.org/

Reply:
cool